### PR TITLE
Added hide password with improved dialog box

### PR DESCRIPTION
**Pull request by shiv07tiwari**
Added hide password with improved dialog box.
OK button wont work if password is null. @Swati4star

### CODE REVIEW

**Comment by Swati4star** [comment]
@shiv07tiwari 
There are checkstyle errors:
```
[ant:checkstyle] [ERROR] /home/travis/build/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java:78:18: Member name 'positiveAction' must match pattern '^m[A-Z][a-zA-Z0-9]*$'. [MemberName]
[ant:checkstyle] [ERROR] /home/travis/build/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java:79:18: Member name 'neutralAction' must match pattern '^m[A-Z][a-zA-Z0-9]*$'. [MemberName]
[ant:checkstyle] [ERROR] /home/travis/build/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java:80:22: Member name 'passwordInput' must match pattern '^m[A-Z][a-zA-Z0-9]*$'. [MemberName]
[ant:checkstyle] [ERROR] /home/travis/build/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java:119: 'array initialization' child have incorrect indentation level 32, expected level should be one of the following: 18, 24, 48, 55. [Indentation]
[ant:checkstyle] [ERROR] /home/travis/build/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java:120: 'array initialization' child have incorrect indentation level 32, expected level should be one of the following: 18, 24, 48, 55. [Indentation]
[ant:checkstyle] [ERROR] /home/travis/build/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java:147: 'array initialization' child have incorrect indentation level 32, expected level should be one of the following: 18, 24, 48, 55. [Indentation]
[ant:checkstyle] [ERROR] /home/travis/build/Swati4star/Images-to-PDF/app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java:148: 'array initialization' child have incorrect indentation level 32, expected level should be one of the following: 18, 24, 48, 55. [Indentation]
```

File: app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java [hunk 1]
```diff
@@ -110,8 +116,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             != PackageManager.PERMISSION_GRANTED)) {
                 mOpenSelectImages = false; // We don't want next activity to open after getting permissions
                 requestPermissions(new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE,
-                                                Manifest.permission.READ_EXTERNAL_STORAGE,
-                                                Manifest.permission.CAMERA},
+                                Manifest.permission.READ_EXTERNAL_STORAGE,
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java]: This change is causing checkstyle error. The indentation level was correct before.

File: app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java [hunk 2]
```diff
@@ -138,8 +144,8 @@ void startAddingImages() {
                     != PackageManager.PERMISSION_GRANTED) {
                 mOpenSelectImages = true; // We want next activity to open after getting permissions
                 requestPermissions(new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE,
-                                                Manifest.permission.READ_EXTERNAL_STORAGE,
-                                                Manifest.permission.CAMERA},
+                                Manifest.permission.READ_EXTERNAL_STORAGE,
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java]: Here also. Indentation problem.

File: app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java [hunk 3]
```diff
@@ -72,6 +75,9 @@
     private String mPath;
     private String mFilename;
     private String mPassword;
+    private View positiveAction;
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java]: change `positiveAction` to `mPositiveAction`
`neutralAction` to `mNeutralAction`
`passwordInput` to `mPasswordInput`

**Comment by shiv07tiwari** [comment]
Fixed indentation problem and renamed the variables. @Swati4star

**Comment by Swati4star** [comment]
Changes look good now. Squash the 2 commits into one now. @shiv07tiwari

**Comment by shiv07tiwari** [comment]
@Swati4star  Squash the 2 commits into one.

**Comment by Swati4star** [comment]
@shiv07tiwari There are 4 commits now.

**Pull request closed without merging**